lib/bcoin.js

```diff
@@ -256,6 +261,70 @@ class LedgerBcoin {
     }
   }
 
+  /**
+   * Nullify other scripts and start sending inputs
+   * @async
+   * @param {MTX} mtx
+   * @param {String} key - Prevout to key
```

Comment:
`prevout to key` does this mean you're accepting a prevout and extracting the key from it?

Reply:
well, prevout is Previous TX Hash + Output index. Key from that is `txhash+index` :)